test/test_time_util.py

```diff
@@ -44,3 +46,230 @@ def test_str_with_microseconds_returns_float(self):
         numeric = time_util.datetime_to_epoch(dt)
         assert (int(numeric) < float(numeric))
         assert (numeric == 1371209423.1)
+
+    def test_discrete_boundaries(self):
+
+        dte = datetime.now()
```

Comment:
Yes, and it can stay that way I think in the test files.

Can you also attach a screenshot for the UI change?